src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java

```diff
@@ -1,14 +1,10 @@
 package uk.gov.hmcts.futurehearings.pip.acceptance.publications;
```

Comment:
Could you please explain the reason(s) to have this test file with no test in it?

Reply:
as above